dataactbroker/routeUtils.py

```diff
@@ -28,6 +28,8 @@ def run_instance_function(accountManager, accountFunction, getSystemEmail = Fals
                 else:
                     # Currently no functions with user but not credentials flag
                     raise ValueError("Invalid combination of flags to run_instance_function")
+            elif(addTrueFlag):
```

Comment:
So the options are either use the flag here or have an additional function that would also call the original error reports function? Am I understanding that correctly?